js/data/move_data.js

```diff
@@ -104,7 +104,8 @@ var MOVES_RBY = {
         bp: 100,
         type: 'Psychic',
         category: 'Special',
-        givesHealth: true
+        givesHealth: true,
+	percentHealed: 0.5
```

Comment:
Now you’re missing a space lol

Reply:
So for some reason the missing space doesn't show up until I publish the commit. Everything looks finely aligned when editing the code.